packages/pwa-kit-runtime/src/ssr/server/express.lambda.test.js

```diff
@@ -323,9 +323,17 @@ describe('SSRServer Lambda integration', () => {
                 path: testCase.path,
                 body: undefined
             })
+            // aws-serverless-express added this header
+            event.headers['x-apigateway-event'] = 'apig-event'
+
+            // Check to make sure the adapter can handle the event
+            const adapter = new ApiGatewayV1Adapter()
+            adapter.canHandle(event)
```

Comment:
Is this required? It looks like it's not assigning it's return value to anything.